### ISSUE TITLE
Remove use of types module constants in twisted/internet

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8451]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)| @rodrigc reported|
|-|-|
|Trac ID|trac#8451|
|Type|enhancement|
|Created|2016-06-12 04:36:50Z|

The *types* module constants have been removed in Python 3.
For example using *types.TupleType* in Python 3 will give an *AttributeError*

These constants should be removed and replaced with the primitive type instead.  See:

http://www.diveintopython3.net/porting-code-to-python-3-with-2to3.html#types

<details><summary>Searchable metadata</summary>

```
trac-id__8451 8451
type__enhancement enhancement
reporter__rodrigc rodrigc
priority__normal normal
milestone__Python_3_x Python-3.x
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1465706210189883 1465706210189883
changetime__1466616132528307 1466616132528307
version__None None
owner__rodrigc rodrigc

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_1"></a>|@rodrigc commented|
|-|-|

See:
https://github.com/twisted/twisted/pull/171

**Comment by twisted-trac** [comment]
|[<img alt="wsanchez's avatar" src="https://avatars.githubusercontent.com/u/50002?s=50" width="50" height="50">](https://github.com/wsanchez)<a name="note_2"></a>|@wsanchez set owner to @rodrigc|
|-|-|

reviewed

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_3"></a>|GitHub set status to `closed`|
|-|-|

In changeset 697fb2dc105d37adcf0aa073657675acd3b5cefb
```
#!CommitTicketReference repository="" revision="697fb2dc105d37adcf0aa073657675acd3b5cefb"
Merge 8451-rodrigc-types: Replace use of types constants with primitive types in twisted/internet

Author: rodrigc
Reviewer: wsanchez
Fixes: #8451
```
